frontends/common/options.cpp

```diff
@@ -292,7 +292,7 @@ FILE* CompilerOptions::preprocess() {
         // line for the preprocessor
         char * driverP4IncludePath =
           isv1() ? getenv("P4C_14_INCLUDE_PATH") : getenv("P4C_16_INCLUDE_PATH");
-        cmd += cstring(" -C -undef -nostdinc") + " " + preprocessor_options
```

Comment:
It seems interesting that here we also use `-C` (preserve comments), `-undef` (remove standard predefines) and `-nostdinc` (remove standard include directories), while in the .cfg files we do not.